examples/my_wallet.json

```diff
@@ -0,0 +1 @@
+{"version":"3.0","accounts":[{"address":"NX1yL5wDx3inK2qUVLRVaqCLUxYnAbv85S","key":"6PYUz1rNSwDf9ad1vxYbJyK93GrnnPBhr819HgSefMvgU1H9QxqVVCZQtN","label":"my_account","contract":{"script":"DCEDhEhWuuSSNuCc7nLsxQhI8nFlt+UfY3oP0/UkYmdH7G5BdHR2qg==","parameters":[{"name":"parameter0","type":"Signature"}],"deployed":false},"lock":false,"isdefault":false}],"scrypt":{"n":16384,"r":8,"p":8},"extra":{"Tokens":null}}
```

Comment:
If we're putting this into `examples`, it should be explained somehow (and the test as well). Previously it was kinda obvious (although not quite for everyone) that we only have subdirectories in `examples` and each of them is an example of its own. Now this file is added to `examples` along with some .go file, so as a user I'm not sure where to start with all of this. Let's create a README here (in `examples`) with very brief introduction to examples and other things we have (and link this README from top-level one).

Reply:
I wrote a README and moved the test to the `pkg/wallet` because this dev-related code inside the `examples` folder may floor the user.